mkdocs.yml

```diff
@@ -66,6 +66,7 @@ nav:
         - Circuits: 'core-functionality/circuits.md'
         - Power Tracking: 'core-functionality/power.md'
         - Tenancy: 'core-functionality/tenancy.md'
+        - Secrets: 'core-functionality/secrets.md'
```

Comment:
We're getting to the point where it makes me wonder if we should alphabetize this list of core functionality items?

Reply:
Sure can do. I don't see any particular logic to the current ordering.